packages/react-instantsearch/examples/react-native/package.json

```diff
@@ -2,15 +2,12 @@
 	"name": "reactnative",
 	"version": "0.0.1",
 	"private": true,
-	"scripts": {
-		"start": "node node_modules/react-native/local-cli/cli.js start"
```

Comment:
It's inside the readme, since it's platform dependant (run-ios or run-android). Works?